R/utils.R

```diff
@@ -2,6 +2,17 @@ last <- function(x) x[length(x)]
 
 sq <- function(x) glue::single_quote(x)
 
+sglue <- function(...) stop(glue::glue(..., .envir = parent.frame()), call. = FALSE)
```

Comment:
👌 will do. He also sets the `.envir` there - it seems strange that you have to but 🤷‍♀️ .